packages/back-end/src/api/sdk-connections/postSdkConnection.ts

```diff
@@ -0,0 +1,80 @@
+import { PostSdkConnectionResponse } from "../../../types/openapi";
+import {
+  toApiSDKConnectionInterface,
+  createSDKConnection,
+} from "../../models/SdkConnectionModel";
+import { createApiRequestHandler } from "../../util/handler";
+import { postSdkConnectionValidator } from "../../validators/openapi";
+import { findAllProjectsByOrganization } from "../../models/ProjectModel";
+import { sdkLanguages } from "../../util/constants";
+import { SDKLanguage } from "../../../types/sdk-connection";
+
+export const postSdkConnection = createApiRequestHandler(
+  postSdkConnectionValidator
+)(
+  async (req): Promise<PostSdkConnectionResponse> => {
+    const {
+      name,
+      languages,
+      sdkVersion,
+      environment,
+      projects = [],
+      encryptPayload = false,
+      includeVisualExperiments = false,
+      includeDraftExperiments = false,
+      includeExperimentNames = false,
+      includeRedirectExperiments = false,
+      proxyEnabled,
+      proxyHost,
+      hashSecureAttributes = false,
+      remoteEvalEnabled,
```

Comment:
Some of these settings are pro/enterprise only.  We should check that here and throw errors if the user doesn't have those commercial features.  Same for the PUT endpoint.

For example:
```ts
if (!req.context.hasPremiumFeature("encrypt-features-endpoint")) {
  // Return error
}
```